app/packages/looker/src/elements/common/util.ts

```diff
@@ -172,17 +172,16 @@ export const getColorFromOptionsPrimitives = ({
     if (setting) {
       // check if this label has a assigned color, use it if it is a valid color
       const valueColor = setting.valueColors?.find((l) => {
-        if (["none", "null", "undefined"].includes(l.value?.toLowerCase())) {
+        const normalized = l.value?.toString().toLowerCase();
+        if (["none", "null", "undefined"].includes(normalized)) {
           return typeof value === "string"
-            ? l.value?.toLowerCase === value
+            ? normalized === value.toLowerCase()
             : !value;
```

Comment:
I'm not sure I understand the intent of returning `!value` here. Shouldn't `value` always be converted to a string to check for a match on `None`/`null`/`undefined`?